sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -130,18 +130,84 @@ public virtual Response<SipTrunk> GetTrunk(
         /// Get <see cref="SipTrunk"/> with provided FQDN.
         /// </summary>
         /// <param name="fqdn">SIP trunk FQDN.</param>
+        /// <param name="expand">Expand enum.</param>
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>Trunk configuration.</returns>
+        /// <exception cref="KeyNotFoundException">Route with specified name wasn't found.</exception>
+        public virtual Response<SipTrunk> GetTrunk(
+            string fqdn,
+            ExpandEnum expand,
```

Comment:
Feedback from review:
don't use ExpandEnum in public methods, let's use:
 boolean includeHealth parameter instead.